src/main/java/com/icthh/xm/ms/entity/domain/idresolver/XmEntityObjectIdResolver.java

```diff
@@ -11,15 +11,19 @@
 import org.springframework.context.annotation.Scope;
 import org.springframework.stereotype.Component;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * XmEntity object ID resolver.
  * Resolves XmEntity object from database bi ID provided in JSON.
  * see https://stackoverflow.com/questions/41989906/jackson-referencing-an-object-as-a-property
  */
+@Slf4j
 @Component
 @Scope("prototype")
-@Slf4j
-public class XmEntityObjectIdResolver extends SimpleObjectIdResolver {
+public class XmEntityObjectIdResolver implements ObjectIdResolver {
+    protected Map<ObjectIdGenerator.IdKey,Object> _items;
```

Comment:
where this map will be cleaned?

Reply:
in the end of scope prototype with object itself